ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19.yaml

```diff
@@ -34,35 +34,6 @@ build_root:
     namespace: ocp
     tag: rhel-9-golang-1.23-openshift-4.19
 images:
-- dockerfile_path: images/Dockerfile.golang
-  from: ci_tests-private-base_4.19
-  inputs:
-    ci_rosa-aws-cli_latest:
-      as:
-      - registry.ci.openshift.org/ci/rosa-aws-cli:latest
-    ci_tests-private-builder_4.18:
-      as:
-      - registry.ci.openshift.org/ci/tests-private-builder:4.18
-    netobserv_network-observability-cli_ci:
-      as:
-      - registry.ci.openshift.org/netobserv/network-observability-cli:ci
-  to: tests-private
-- dockerfile_literal: |
-    FROM src AS builder
-    FROM registry.ci.openshift.org/ci/tests-private-baseui:4.19
-    RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm && \
-        yum install -y ./google-chrome-stable_current_*.rpm && \
-        mkdir -p /go/src/github.com/openshift
-    COPY --from=builder /go/src/github.com/openshift/openshift-tests-private /go/src/github.com/openshift/openshift-tests-private
-    WORKDIR /go/src/github.com/openshift/openshift-tests-private
-  inputs:
-    ci_tests-private-baseui_4.19:
-      as:
-      - registry.ci.openshift.org/ci/tests-private-baseui:4.19
-    src:
-      as:
-      - src
-  to: tests-private-ui
```

Comment:
`tests-private` and `tests-private-ui` are removed here because they are duplicate with https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__images.yaml#L40-L68
```
time="2025-04-14T23:47:31Z" level=error error="output tag ci/tests-private-ui:4.19 is promoted from more than one place: openshift/openshift-tests-private@release-4.19, openshift/openshift-tests-private@release-4.19 [images]"
time="2025-04-14T23:47:31Z" level=error error="output tag ci/tests-private:4.19 is promoted from more than one place: openshift/openshift-tests-private@release-4.19, openshift/openshift-tests-private@release-4.19 [images]"
```